gamemode/modules/party/cl_party.lua

```diff
@@ -0,0 +1,162 @@
+RVR.Party = RVR.Party or {}
+local party = RVR.Party
+
+function party.tryCreateParty( name, tag, color, joinMode, callback )
+    if joinMode < 0 or joinMode > 2 then return end
```

Comment:
could you use the joinMode constants here or perhaps the length of `party.joinModeStrs`

Reply:
I could do `if not party.joinModeStrs[joinMode] then`?

Reply:
that would be better